### DISCUSSION TITLE
Testing Redash on WAMP server application

### DISCUSSION THREAD

**Discussion by Brada65** [getredash/redash#6956] [Q&A]
Can Redash be installed on Windows using the WAMP server application for testing purposes? 
It looks easier to use than Grafana, the only problem is that Grafana has Windows installation, and Redash does not.
I need to test Business Intelligence tools for my organization and because we are a non-profit only open-source (free) tools are an option. Again, it looks nice that I can write queries and choose the graphs to create dashboards for analytics and reporting, but I need to do the testing and show the management that can be run on our server with data kept within our network.
I appreciate any help you can provide.

**Comment by robbohn** [comment]
I've been running Redash on WSL as a non-production prototype with no issues, connecting to several SQL Servers.  I'm running Ubuntu 2204 on Windows 10 using WSL2, and I used the standard redash setup script.

**Reply by justinclift** [reply]
Cool, that's good info.

What's the approach you're taking for backing up the database and matching environment file (it has the encryption keys and database password in it)?

Backing up stuff when installed on a Linux box is pretty straight forward (for me, due to familiarity), but I'm not personally familiar with any standard Windows backup tools. :smile:
